### DISCUSSION TITLE
Seems like it could automatically add the "Notifer" suffix?

### DISCUSSION THREAD

**Discussion by MyklClason** [excid3/noticed#491] [Ideas]
Coming from ViewComponents, it's rather nice that we can create components without having to always add the "Component" part of the class name.

I want to request the same functionality here, with a config setting to handle it.

This would allow us to create notifiers via:

```
rails generate noticed:notifier NewComment
```

or

```
rails generate noticed:notifier new_comment
```

or

```
rails generate noticed:notifier comments/new # Comment::NewNotifier
```

**Comment by excid3** [comment]
It already does. https://github.com/excid3/noticed/blob/main/lib/generators/noticed/templates/notifier.rb.tt#L5
